### PR TITLE
Preflight verbiage for RHEL & OpenStack certs

### PR DESCRIPTION
**Pull request by jfrancin**
Add a sentence about how Preflight is required for OpenStack and RHEL certs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 requirements for [Red Hat OpenShift
 Certification](https://connect.redhat.com/en/partner-with-us/red-hat-openshift-certification).
 
+Running Preflight is also a required step in certifying all containers that are part of an [OpenStack](https://www.redhat.com/en/technologies/linux-platforms/openstack-platform) or [Red Hat Enterprise Linux](https://connect.redhat.com/partner-with-us/red-hat-enterprise-linux-certification) partner product.
```
Comment (acornett21) [inline README.md:11]: I don't know if I love just tacking on another block for this. I feel like the first sentence should be updated to encompass all certification that preflight is apart of, I just don't know what that wording would be.

Comment (jfrancin) [reply README.md:11]: sigh.  I've found that with some partners - especially ones for whom English is not their first language - that they would stop reading after the first sentence in a paragraph if they think it doesn't apply to them.  I could re-work this into a bulleted list - perhaps something along the lines of:

----- cut here -----
**Preflight** is a command line (CLI) tool to verify that partner-submitted containers meet minimum requirements for Red Hat Software Certification. These include:

- [OpenShift](https://www.openshift.com) Operator Bundles
- OpenShift containers
- [OpenStack](https://www.redhat.com/en/technologies/linux-platforms/openstack-platform) containers
- [Red Hat Enterprise Linux](https://connect.redhat.com/partner-with-us/red-hat-enterprise-linux-certification) application containers

Comment (acornett21) [reply README.md:11]: I think that's a cleaner way to go.

Comment (jfrancin) [reply README.md:11]: Very well - I'll change it.

**Pull request closed without merging**